comp/volunteers/inform-volunteers.yaml

```
@@ -1,6 +1,6 @@
-summary: Prepare and send out volunteer information pack
+summary: Inform volunteers for the $SRYYYY competition
 
-priority: major
+priority: should
```

Question: Why was this downgraded?

```suggestion
priority: must
```